app/controllers/course/forum/posts_controller.rb

```diff
@@ -0,0 +1,56 @@
+class Course::Forum::PostsController < Course::Forum::Controller
+  include Course::Discussion::PostsConcern
+
```

Comment:
yes, I noticed this problem. Then I think I need to change the ability to `:reply`. Means if you can reply to a topic then you can use post related methods.